### ISSUE TITLE
revert got very slow (factor 37 to 3.0 release)

### ISSUE THREAD

**Issue by codingdave** [darktable-org/darktable#5779]
**Describe the bug**
reverting an image takes a lot of time in current master. The speed regression is so big that hovering the mouse away from an image after having pressed r does not revert that image any longer.

**To Reproduce**
1. have latest master started with -d all
2. Have culling mode with 1 image and no sidebars (<TAB>)
3. observe the timings of reject on the console. On my machine I can observe a factor of 37, with darktable-3-0 ~0.04 seconds and current master ~1.51 seconds per reject.

**Expected behavior**
rejecting images should be as snappy as it was previously

**Platform (please complete the following information):**
darktable 3.1.0+2483~g3e898ae53

**Additional context**
Looking at the console much more stuff is happening with latest master than it was with 3.0. Even moving the cursor triggers dt_history_hash_get_status() calls. 

My images are on an internal SSD. 

The speed regression was observed on the same machine I have available master and 3.0.

**Comment by johnny-bit** [comment]
While for me "reject" in culling mode is snappy AF I can still confirm that dt_history_hash_get_status is called even wehn mouse just moves over image (it is not called for thumbnails in file manager mode).

If you still have speed problems, let's investigate... if not, maybe just close (or edit) this issue to focus solelly on unneded calls to dt_history_hash_get_status that may slow down stuff and ping @AlicVB :)

**Comment by codingdave** [comment]
I have tested today with  darktable 3.5.0+538~g1b0cc6c0e and this call:
~ ~/unstable/darktable_master/bin/darktable --configdir /tmp/ --library /tmp/sdf -d all

I have imported a few hundred images.
I am not able to reproduce the slow reject behavior that I have reported half a year ago.
However, this is with a clean database on another computer. It might be the amount of files that affects the reject speed, it might be that files are mounted via nfs on the other computer. I will close this issue now and re-open if the issue is still present on the other system.
